Sources/ArcGISToolkit/FloorFilter/FloorFilterViewController.swift

```diff
@@ -0,0 +1,540 @@
+        
+        // Adjust the constraints and order of the views in the Floor Filter if placement is on top or bottom of the sceen
+        adjustConstraintsBasedOnPlacement()
+    }
+    
+    private func initializeFloorManager() {
+        guard let floorManager = floorManager else { return }
+        
+        DispatchQueue.main.async {
+            floorManager.load(completion: { error in
+                if (error != nil || floorManager.loadStatus != .loaded) {
+                    return
+                }
+                
+                if (floorManager.loadStatus == .loaded) {
+                    self.viewModel.reset()
+                    self.viewModel.floorManager = floorManager
+                    self.initializeSiteButton()
+                    
+                    // Filter the map to any previously selected level
+                    self.viewModel.filterMapToSelectedLevel()
+                    self.levelsTableView.reloadData()
+                }
+            })
+        }
+    }
+    
+    private func initializeSiteButton() {
+        // Disable the Site Button if the Floor Manager is not loaded yet
+        if (viewModel.floorManager == nil) {
+            siteBtn.backgroundColor = UIColor.systemGray3
+            siteBtn.isUserInteractionEnabled = false
+        } else {
+            addShadow()
+            siteBtn.backgroundColor = backgroundColor.withAlphaComponent(0.9)
+            siteBtn.isUserInteractionEnabled = true
+        }
+    }
+    
+    private func initializeButtonsClickListeners() {
+        siteBtn.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(self.showSiteFacilityPrompt)))
+        siteBtn.isUserInteractionEnabled = true
+        
+        closeBtn.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(self.collapseLevelsList)))
+        closeBtn.isUserInteractionEnabled = true
+    }
+    
+    private func initializeLevelsTableView() {
+        levelsTableView.delegate = self
+        levelsTableView.dataSource = self
+        levelsTableView.separatorStyle = .none
+    }
+    
+    @objc func showSiteFacilityPrompt(sender: UITapGestureRecognizer) {
+        let storyboard = UIStoryboard(name: "FloorFilter", bundle: .module)
+        
+        if let siteFacilityPromptVC = storyboard.instantiateViewController(identifier: "SiteFacilityPromptVC") as? SiteFacilityPromptViewController {
+            siteFacilityPromptVC.providesPresentationContextTransitionStyle = true
+            siteFacilityPromptVC.definesPresentationContext = true
+            siteFacilityPromptVC.modalPresentationStyle = UIModalPresentationStyle.overCurrentContext
+            siteFacilityPromptVC.modalTransitionStyle = UIModalTransitionStyle.coverVertical
+            
+            if let topVC = UIApplication.shared.topMostViewController() {
+                topVC.present(siteFacilityPromptVC, animated: true, completion: nil)
+            }
```

Comment:
I think this can be simplified:
```suggestion
            siteFacilityPromptVC.modalPresentationStyle = .automatic
            present(siteFacilityPromptVC, animated: true)
```